ui-admin/src/portal/media/SiteMediaDeleteModal.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react'
+import { Modal } from 'react-bootstrap'
+import Api, { SiteMediaMetadata } from 'api/api'
+import { doApiLoad } from 'api/api-utils'
+import LoadingSpinner from 'util/LoadingSpinner'
+import { successNotification } from 'util/notifications'
+import { Store } from 'react-notifications-component'
+import { Button } from 'components/forms/Button'
+import { LoadedPortalContextT } from '../PortalProvider'
+
+/** Renders a modal for an admin to submit a sample collection kit request. */
+export default function SiteMediaDeleteModal({
+  portalContext,
+  onDismiss,
+  onSubmit,
+  media
+}: {
+    portalContext: LoadedPortalContextT,
+    onDismiss: () => void,
+    media: SiteMediaMetadata,
+    onSubmit: () => void }) {
+  const [isLoading, setIsLoading] = useState<boolean>(false)
+  const deleteImage = () => {
+    doApiLoad(async () => {
+      await Api.deletePortalMedia(portalContext.portal.shortcode, media.id)
+      Store.addNotification(successNotification('file successfully deleted'))
+      onSubmit()
+    }, { setIsLoading })
+  }
+
+  return <Modal show={true} onHide={onDismiss}>
+    <Modal.Header closeButton>
+      <Modal.Title>Delete Media</Modal.Title>
+    </Modal.Header>
+    <Modal.Body>
```

Comment:
BLOCKER: This is an irreversible delete and we don't have assurances that the image isn't being used on the 'live' site.  I think until we develop that check (which won't be trivial, since it will involve parsing a lot of SiteContent objects and so is way outside the scope of this ticket), we should restrict deletes to superusers only.  For non-superusers, this dialog can just tell them to email support and include [[url of medial to delete]]"